node/src/actors/chain_manager/mod.rs

```diff
@@ -1058,27 +1058,28 @@ fn update_reputation(
     {
         log::error!("Error updating reputation in consolidation: {}", e);
     }
+    if log_level != log::Level::Trace {
```

Comment:
We can also skip this if the current log level is INFO or ERROR, so try doing:
```suggestion
    if log_level >= log::max_level() {
```

Reply:
Even better:
```suggestion
    if log_enabled!(log_level) {
```